### PR TITLE
gui: Add a `Make unsigned` button next to `Send`

### PR DESCRIPTION
**Pull request by achow101**
Instead of having the Send button mutate into a Create Unsigned button when `IsWalletFlagSet(WALLET_FLAG_DISABLE_PRIVATE_KEYS)`, have a second button that always creates a psbt. The original Send button will then not change its behavior and just disable itself when private keys are disabled.

#17509 does a refactor that makes this easier, so this PR is dependent on that.

Split from #18627

### CODE REVIEW

File: src/qt/forms/sendcoinsdialog.ui [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>850</width>
-    <height>526</height>
+    <height>572</height>
```
Comment (jonasschnelli) [inline src/qt/forms/sendcoinsdialog.ui:10]: I would clean out the unnecessary changes in the .ui file (usually done by Qt Creator).

Comment (achow101) [reply src/qt/forms/sendcoinsdialog.ui]: Done

**Review by promag — COMMENTED** [review]
Concept ACK.

**Comment by luke-jr** [comment]
This would likely confuse users as-is.

Can we simplify the workflow so it always makes a PSBT, but in a way that users can easily sign+broadcast?

Perhaps the PSBT dialog can become like the current send confirmation dialog unless an expert-mode option is enabled?

**Comment by instagibbs** [comment]
I think sticking this behind an expert mode switch is probably the right thing to do at a minimum.

**Comment by achow101** [comment]
> Can we simplify the workflow so it always makes a PSBT, but in a way that users can easily sign+broadcast?

I've simplified this workflow by changing `SendConfirmationDialog` instead. This has been unified with #18655 in #18789.
